ui/apps/dashboard/src/app/(dashboard)/env/EnvironmentListTable.tsx

```diff
@@ -66,27 +81,42 @@ export default function EnvironmentListTable({ envs }: { envs: Environment[] })
           </th>
 
           <th scope="col" className="w-0 pr-4"></th>
-
-          {/* TODO - When we have this data bring back this column header here and in TableRow */}
-          {/* <th
-                    scope="col"
-                    className="font-medium py-3 px-4 text-slate-500 text-sm text-right"
-                  >
-                    Latest Deployment
-                  </th> */}
         </tr>
       </thead>
       <tbody className="divide-y divide-slate-100 px-4 py-3">
-        {envs.length ? (
-          envs.map((env, i) => <TableRow env={env} key={i} />)
+        {envs.length === 0 ? (
+          <tr>
+            <td colSpan={5} className="px-4 py-4 text-center text-sm font-semibold text-slate-500">
+              There are no actively synced branches
```

Comment:
Not a blocker, but the phrasing in "actively synced branches" or the previous "actively deployed branches" seems difficult to read